src/schematools/factories.py

```diff
@@ -407,6 +408,36 @@ def _build_geo_indexes(
     ]
 
 
+def _build_temporal_indexes(
```

Comment:
If possible, remove the leading `_`, because now we are using an internal attribute of the class which is considered dangerous.

Reply:
Done